packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -89,19 +89,19 @@ export class RedirectsMiddleware {
     const hostname = this.getHostname(req);
```

Comment:
Add unit tests to cover new case, when response is provided

Reply:
done, two tests added